.github/workflows/test-template.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       cancel-in-progress: false
     outputs:
       cache_key: ${{ env.SWIFT_IMAGE }}-vapor-toolbox-${{ steps.latest.outputs.tag }}
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Updating the container makes sense, but changing the `runs-on` will just lock us in to a version instead of moving ahead when GH updates the `-latest` tag.